app/build.gradle

```diff
@@ -22,6 +22,7 @@ android {
     buildFeatures {
         viewBinding true
     }
+    namespace 'com.tealium.demo'
 }
 
 dependencies {
```

Comment:
Might be worth updating the Remote Command and Tag Mgmt Dispatchers to latest versions here.